packages/web-components/src/tree-view/tree-view.style.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import { css } from '@microsoft/fast-element';
+import { spacingHorizontalNone } from '../theme/design-tokens.js';
+
+export const styles = css`
+  :host {
+    /* Set the font size of the tree view to 0 so we can calculate the indentation based on it */
+    font-size: 0;
+    outline: none;
+  }
+  /* The direct child treeitems of the tree-view component should not get the padding */
+  ::slotted([role='treeitem']) {
```

Comment:
Just a note that if we move this to element internals we'll need to find a different selector. In the past, we've supported _any_ variations of an element so long as it matches the role, though I'm not sure we need to do that anymore given the goals of Fluent.